crates/sncast/src/response/structs.rs

```diff
@@ -144,3 +144,17 @@ pub struct TransactionStatusResponse {
 }
 
 impl CommandResponse for TransactionStatusResponse {}
+
+#[derive(Serialize)]
+pub enum VerificationStatus {
+    OK,
+    Error,
+}
+
+#[derive(Serialize)]
+pub struct VerifyResponse {
+    pub verification_status: VerificationStatus,
+    pub message: Option<String>,
+}
```

Comment:
Should be probably an alias of `Result<String>` - i think that's the intent here?